src/lib/zexe_backend/zexe_backend_common/dune

```diff
@@ -14,4 +14,5 @@
     snarky.backendless
     sponge
     snarkette
+    coda_version
```

Comment:
This adds a transitive dependency to everything above this library on `universe`. I can't think of any easy fix for it, but we should try to come up with one ASAP, or push forward with moving these things out of the build process altogether.

Reply:
It's not ideal, but if we change the zexe interface so that we thread the marlin repo sha down to it instead of baking it in at that layer, we could hack around this. It's not a great solution imo, but it would fix rebuilding zexe_backend every single build, which seems worth it.

Reply:
I may be able to do a hack where I depend on the source tree of the marlin repo in external and generate a file internally to zexe_backend that contains the sha for it. If I do it right, it will only trigger a rebuild when the marlin repo changes, but we'll see if I can get dune to play nice like that. I'll test it out some before I merge this.